tests/test_user_confirm_invites_and_register.py

```diff
@@ -172,6 +172,66 @@ def test_register_weak_password(registry_form_data, client):
     assert user is None
 
 
+def test_register_nonlatin1_username(registry_form_data, client):
+    """Username can only contain latin 1 encodable characters.
+
+    Requests package does not allow non-latin1 encodable characters.
+    """
+    form_data = registry_form_data
+
+    form_data["username"] = "user_€"  # € is invalid
+
+    # Request should work
+    response = client.post(
+        tests.DDSEndpoint.USER_NEW,
+        json=form_data,
+        follow_redirects=True,
+        headers=tests.DEFAULT_HEADER,
+    )
+    assert response.status == "200 OK"
```

Comment:
is there any particular reason that the status check is done line this at some places:
> assert response.status_code == http.HTTPStatus.OK
and like this at other:
> assert response.status == "200 OK"

I mean, for consistency reasons we should probably stick to one of them

Reply:
No particular reason, just some copy paste and then changes elsewhere in code. Will fix